Gemfile

```diff
@@ -35,7 +35,7 @@ group :development do
   gem "rubocop-performance", '= 1.16.0',         require: false
   gem "rubocop-rspec", '= 2.19.0',               require: false
   gem "rb-readline", '= 0.5.5',                  require: false, platforms: [:mswin, :mingw, :x64_mingw]
-  gem "bolt", '>= 3.10.0',                       require: false
+  gem "bolt", '>= 3.27.2',                       require: false
```

Comment:
Why is bolt updated? Do old versions not work anymore with PEADM? That's not documented in the backup_restore.md nor the README.md

Reply:
We felt that it was good to increment the version to keep it within the last couple of years. Not that is wont work anymore, but just difficult for us to sign off support for every version.